README.md

```diff
@@ -1,10 +1,129 @@
 # GenZ
-## Requirements
 
-- [Go >1.20](https://go.dev/doc/install)
-## Try it out
-Explore built-in `examples`, clone repo, and run
+[![Go Reference](https://pkg.go.dev/badge/github.com/Joffref/genz.svg)](https://pkg.go.dev/github.com/Joffref/genz)
+[![Go Report Card](https://goreportcard.com/badge/github.com/Joffref/genz)](https://goreportcard.com/report/github.com/Joffref/genz)
+[![License](https://img.shields.io/github/license/Joffref/genz)](LICENSE)
+![GitHub stars](https://img.shields.io/github/stars/Joffref/genz)
+
+> At this moment, this project is in development. It is not ready for production use. See the [Contributing](#contributing) section if you would like to help make it better.
+
+✨ Never write toil/boilerplate code again! ✨
+
+Thanks to GenZ, you can generate code for your own needs, without having to write a single line of code.
```

Comment:
Technically... our templates are a superset of Go code, and there will be Go tests, but it's okay !